src/components/viewsComponents/events/constants/EventsData.ts

```diff
@@ -9,48 +9,50 @@ export interface Event {
   allDay: boolean;
 }
 
-export const staticEvents: Event[] = [
+const formatDate = (date: Date): string => format(date, "yyyy-MM-dd'T'HH:mm");
```

Comment:
function inside constants ? do we even need this function even ? I think it should be show like in all other components that use it

Reply:
Changed type of IEvent properties to 'Date' and now only parsing to string inside component if needed.